src/components/EditImageModal/index.jsx

```diff
@@ -404,10 +529,46 @@ export default class EditImageModal extends React.Component {
       imageSource: event.detail.src || '',
       imageStyle: event.detail.style || '',
       isImageDecorative: event.detail.alt === '',
+      // if existing img had a source, assume it could be loaded and show the image preview
+      isImageLoaded: !!event.detail.src,
```

Comment:
Doesn't the `!!` cancel itself out? Could we just remove those?

Reply:
Nevermind :) Picked @arizzitano's brain

Reply:
Yeah, it's a weird thing :P.